packages/blaze/view.js

```diff
@@ -184,6 +184,34 @@ Blaze.View.prototype.autorun = function (f, _inViewScope) {
   return c;
 };
 
+/**
+ * Just like Blaze.View#autorun, but with Meteor.subscribe instead of
+ * Tracker.autorun. Stop the subscription when the view is destroyed.
+ * @return {SubscriptionHandle} A handle to the subscription so that you can
+ * see if it is ready, or stop it manually
+ */
+Blaze.View.prototype.subscribe = function (/* arguments */) {
+  var self = this;
+
+  if (! self.isCreated) {
+    throw new Error("View#subscribe must be called from the created callback at the earliest");
+  }
+  if (this._isInRender) {
```

Comment:
This is a typical pattern in the Blaze code, but maybe it should be moved into a helper.